config/initializers/sidekiq.rb

```diff
@@ -9,7 +9,8 @@
 Redis.current = Redis.new(redis_conf)
 
 redis_settings = {url: Redis.current.client.id,
-                  namespace: redis_conf["namespace"],}
+                  namespace: redis_conf["namespace"],
+                  id: nil,}
```

Comment:
This fixes an issue I was running into locally with specs:  https://github.com/mperham/sidekiq/issues/3518#issuecomment-390898217